Makefile

```diff
@@ -4,4 +4,13 @@ debug:
 
 .PHONY: test
 test: 
-	go test -race -cover ./...
\ No newline at end of file
+	#LIBRARY_TEST=TRUE is used by dp-cookies/cookies/cookies.go @ L37 to identify whether its running in a test/locally, as we don't have the means to test secure cookies.
+	LIBRARY_TEST=TRUE go test -race -cover ./...
```

Comment:
hmmm but now this is set to true _always_, even when doing `make test` in a non-local environment. 

I'm not sure I understand what is happening 😅

Reply:
So it will set LIBRARY_TEST to true when its running tests.  The previous name for the environmental variable I thought was misleading, as it needs to be set true when running in concourse pipeline.  Clearer?

Reply:
Oh I assumed it needed to be false when runnin tests concourse. I think I'm still confused 😅

Reply:
Shall we have a chat about this one?